source/WikiSource/Home.md

```diff
@@ -0,0 +1,56 @@
+# Welcome to the SqlServerDsc wiki
```

Comment:
Should be "GPRegistryPolicyDsc" instead of "SqlServerDsc"?

Reply:
Thanks. Will fix.